packages/babel-helper-get-function-arity/README.md

```diff
@@ -1,5 +1,19 @@
 # @babel/helper-get-function-arity
 
+Function that returns the number of arguments or operands that a function takes.
```

Comment:
Im not sure what did u mean by operands here. Would be good to clarify it, but I think saying that it returns number of arguments is quite enough + clarification on what is considered to be an argument, for that we can just send readers to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/length

Reply:
Sounds good.  The operands part was taken from the wikipedia definition.  Will make changes.